lib/constants/platforms.ts

```diff
@@ -3,6 +3,7 @@ export type PlatformId =
   | 'codecommit'
   | 'bitbucket'
   | 'bitbucket-server'
+  | 'gerrit'
```

Comment:
Please put Gerrit in the list of supported platforms in the Renovate `readme.md`, with a link to the Renovate platform docs for Gerrit: :wink: 

- https://github.com/Onesty-Tech-GmbH/renovate/blob/gerrit_v3/readme.md#supported-platforms